.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +10,16 @@ Please include
 ## Todos
 - [ ] Tests
 - [ ] Documentation
+- [ ] Release note
 
+## Release Note
+<!-- Writing a release note:
+- By default, no release note action is required.
+- If you're unsure whether or not your PR needs a note, ask your reviewer for guidance.
+- If this PR requires a release note, update the block below to include a concise note describing
+  the change and any important impacts this PR may have.
+-->
+
+```
```

Comment:
I noticed the k8s one uses a marker string here so they can collect the notes with a bot or more easily search for it.  Might be worth doing?